Resources/Styles/ControlStyles.xaml

```diff
@@ -629,4 +629,133 @@
         <Setter Property="Margin" Value="10"/>
         <Setter Property="HorizontalAlignment" Value="Left"/>
     </Style>
+
+    <Style x:Key="ChequeViewFillInFormStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="TextBlock.FontFamily" Value="Sora" />
```

Comment:
Its target type already defined as TextBlock, so it is redundant to set Property with `FontFamily` under TextBlock keywords again. Can you took out the TextBlock and just left the `FontFamily` ?